### ISSUE TITLE
Brazilian Portuguese Translation

### ISSUE THREAD

**Issue by andrescalco** [getify/Functional-Light-JS#152]
Hello, I would like to translate this book into Brazilian Portuguese.

**Comment by getify** [comment]
Great! See this thread for steps forward: https://github.com/getify/Functional-Light-JS/issues/115

**Comment by evansantos** [comment]
@andrescalco did you started it? how can I help?

**Comment by khullah** [comment]
I can help as well

**Comment by anabastos** [comment]
Hi guys. We're working on a Brazilian Portuguese translation right here: https://github.com/lambda-study-group/Functional-Light-JS

**Comment by khullah** [comment]
@anabastos, como saber o que está traduzido e o que não está? Ou se alguém já "pegou" uma parte para si? Enfim, qual o fluxo?

Acho que no README.md poderia ter uma referencia para isso, ou uma indicação para um CONTRIBUTING.md especial da tradução em si

**Comment by anabastos** [comment]
@khullah That's true. We just added a section at README.md specifying how people can collaborate translating 😸
